### PR TITLE
CodeCamp express importer as generic Designer extension 4 the Market

### PR DESCRIPTION
**Pull request by ivy-rew**
treasure @ivy-bb ' s efforts and keep am ready to present for LTS 10 :guitar:

### CODE REVIEW

File: express-importer-product/pom.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.axonivy.market</groupId>
```
Comment (alexsuter) [inline express-importer-product/pom.xml]: groupid

File: express-importer/pom.xml [hunk 2]
```diff
@@ -0,0 +1,172 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.axonivy.portal.util</groupId>
```
Comment (alexsuter) [inline express-importer/pom.xml]: this is the namespace of portal

File: express-importer/src/ch/ivyteam/ivy/designer/process/ui/internal/express/ExpressFormElement.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
+package ch.ivyteam.ivy.designer.process.ui.internal.express;
```
Comment (alexsuter) [inline express-importer/src/ch/ivyteam/ivy/designer/process/ui/internal/express/ExpressFormElement.java]: namepsace should be reflected in the package names

File: express-importer/pom.xml (hunk 2 above)
Comment (ivy-rew) [reply express-importer/pom.xml]: I added the "util" part? But I'm open for a better ID . Any suggestions?
Maybe `com.axonivy.portal.community.util` ?

File: express-importer-product/pom.xml (hunk 1 above)
Comment (ivy-rew) [reply express-importer-product/pom.xml]: oops :relaxed:

File: express-importer/pom.xml (hunk 2 above)
Comment (alexsuter) [reply express-importer/pom.xml]: all new portal projects:
<groupId>com.axonivy.portal</groupId>

doc-factory
  <groupId>com.axonivy.utils.docfactory</groupId>


proposal
<groupId>com.axonivy.portal.express</groupId>

someday. express and express importer should live in it's same repository. there is a high coupling.

maybe it should not have portal in the namepsace. but then we can change both.

**Pull request merged**